Sources/Plasma/FeatureLib/pfMoviePlayer/plMoviePlayer.h

```diff
@@ -55,6 +55,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 class plMessage;
 
+#ifdef USE_WEBM
+#   include <vpx/vpx_decoder.h>
+#endif
+
```

Comment:
Can we use a forward declaration instead of exposing libvpx to consumers of `plMoviePlayer`?

Reply:
I tried but vpx_image_t is a struct, which AFAIK can't be forward declared. I could anonymize it as a void * though.

Reply:
`typedef struct vpx_image vpx_image_t;` is probably what you're looking for.

Reply:
That worked!